source/puppet/4.3/reference/puppet_collections.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: default
+title: "About Puppet Collections and Packages"
+canonical: "/puppet/latest/reference/puppet_collections.html"
+---
+
+{% include puppet-collections/_puppet_collections_intro.md %}
```

Comment:
The includes from here to the "Verifying Puppet packages" heading supplant the Package Repositories guide content. When adding this to prior versions, we can use the granular includes to only list operating systems compatible with that version of Puppet.

I went with this approach as Puppet Collections have (and I imagine will continue to) span multiple Y or X versions of Puppet and related tools. By breaking these includes down to small pieces, we can tailor this document with a minimum of prose editing, update it from a central location as new Puppet Collections are released or existing PCs are modified, and reuse its content across other documents such as the versioned installation and upgrade content.